scripts/install_redis_cli.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+# Copyright 2016 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+##########################################################################
+
+# INSTRUCTIONS:
+#
+# This script sets up the redis-cli for a local redis caching
+# service that can be used to cache objects in a local development environment.
+# Run the script from the oppia root folder:
+#
+#   bash scripts/install_redis_cli.sh
+#
+# Note that the root folder MUST be named 'oppia'.
+
+cd third_party/redis-cli-6.0.6 &&
+make distclean &&
```

Comment:
Unfortunately I didn't find a built in version. The node version is not viable since it is a packaged version of the redis-cli not redis-server which is the one that we need. The redis cli is a tool to communicate with any redis server. The redis server is a local developmental server that we can use as a testing server for local development. 

Sean and I talked about it and for now we will just built it locally in the third_party directory using make and possibly use a specially uploaded built version in the future.